.gitignore

```diff
@@ -77,3 +77,6 @@ access.log
 \.idea/markdown-navigator\.xml
 
 \.idea/markdown-navigator/profiles_settings\.xml
+certs/PKI-ROOT-CA.cer
```

Comment:
It would probably be better to exclude the entire `certs/` directory here instead of specific files to avoid inadvertently committing such a file if another is created for whatever reason.

Reply:
Thanks Lea, not sure how this got in there but will definitely take that out 👍

Reply:
This has now been changed